### ISSUE TITLE
Action bias is added twice in TD3 algorithm implementation

### ISSUE THREAD

**Issue by implausibleDeniability** [vwxyzjn/cleanrl#259]
## Problem Description

In the implementation of [TD3 algorithm](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/td3_continuous_action.py), the action bias is added twice to the action. First, in the [line 180](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/td3_continuous_action.py#L180) it is added during actor forward pass. Second, in the [line 181](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/td3_continuous_action.py#L181) the random noise comes from a distribution with the center in `actor.action_bias`.

## Possible Solution
I assume rewriting the 181th line to 
```
actions += torch.normal(0, actor.action_scale * args.exploration_noise)
```
 will solve the problem.

**Comment by dosssman** [comment]
@implausibleDeniability Thanks for the catch. In hindsight, it should have been centered at 0 indeed.

**Comment by vwxyzjn** [comment]
Closed by #260. Thanks @implausibleDeniability for raising this issue.
